code/__HELPERS/reagents.dm

```diff
@@ -72,3 +72,14 @@
 	if(!GLOB.chemical_reactions_list[primary_reagent])
 		GLOB.chemical_reactions_list[primary_reagent] = list()
 	GLOB.chemical_reactions_list[primary_reagent] += R
+
+/datum/reagents/proc/create_foam(foam_volume,metaltype = 0,notification = null)
+	var/location = get_turf(my_atom)
+	var/datum/effect_system/foam_spread/foam = new()
```

Comment:
You always create the same type now ?

Reply:
I think it functionally works the same now that the metal foam set_up was rolled into the regular one. If it doesn't I'll have to change back, although I probably should anyways for modularity's sake (I don't know where the resin foam code is but this should probably be there)

Reply:
Both /metal and /metal/smart create different foam types so it's not the same.